app/Http/Controllers/PollController.php

```diff
@@ -10,6 +10,13 @@ class PollController extends Controller
 {
     public function create(Request $request): JsonResponse
     {
+        $request->validate([
+            'name' => 'required|max:255|distinct',
```

Comment:
[unique:table,column](https://laravel.com/docs/9.x/validation#rule-unique)

instead of `distinct`